### PR TITLE
Download and verify top-level targets

### PR DESCRIPTION
**Pull request by trishankatdatadog**
Partial remedies for #7 and #13.

The failure is expected considering only top-level but not delegated targets work.

### CODE REVIEW

File: src/tuf_on_a_plane/config.py [hunk 1]
```diff
@@ -27,13 +28,13 @@ class Config:
 
     # Maximum number of root rotations.
     MAX_ROOT_ROTATIONS = 2 ** 5
-    # 16K ought to be more than enough for everyone ;)
-    MAX_ROOT_LENGTH: Length = Length(2 ** 14)
+    MAX_ROOT_LENGTH: Length = Length(2 ** 15)
```
Comment (trishankatdatadog) [inline src/tuf_on_a_plane/config.py:31]: Now that we check the upper bound on all downloads, there is an issue I don't understand right now where root and timestamp were somehow still slightly bigger than we expected. Will look into this later.

**Comment by trishankatdatadog** [comment]
Completely optional, but would @joshuagl @sechkova @MVrachev @jku be interested in quickly reviewing? 🙂

File: src/tuf_on_a_plane/config.py (hunk 1 above)
Comment (trishankatdatadog) [reply src/tuf_on_a_plane/config.py:31]: Fixed with 2f1e818d4c28662ba971344fe94a2fdb2883b9c1

**Review by juliendoutre — APPROVED** [review]
Looks good to me! I like the refactor of checks in dedicated methods and the use of list and dict comprehension instead of while loops.

**Pull request merged**